supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -1,7 +1,7 @@
 [discrete]
 [[backtrace]]
 ==== image:images/yes.png[yes] backtrace (noun)
-*Description*: "Backtrace" is another name for "stack trace" (or "stack backtrace"), which is a report of the active stack frames (function calls) at a certain point in time during the execution of a program. The Python programming language uses the term "traceback," possibly because the stack frames are printed in the opposite order of those presented by gdb, the GNU Debugger. "Traceback" is the preferred term when referring to a Python stack trace.
+*Description*: "Backtrace" is another name for "stack trace" (or "stack backtrace"), which is a report of the active stack frames (function calls) at a certain point in time during the execution of a program. The Python programming language uses the term "traceback", possibly because the stack frames are printed in the opposite order of those presented by gdb, the GNU Debugger. "Traceback" is the preferred term when referring to a Python stack trace.
```

Comment:
@bredamc I think commas and periods always go inside the quotes for American grammar.
https://www.grammarly.com/blog/quotation-marks/#:~:text=In%20the%20United%20States%2C%20the,sure%20that's%20exactly%20what%20happened.%E2%80%9D

Reply:
Thanks so much for reviewing for me, @joaedwar! The change I suggested is to follow the advice in IBM Style.